Components/navbar.css

```diff
@@ -0,0 +1,40 @@
+.navbar {
+    background-color: var(--primary-red);
+}
+
+.nav-menu button {
+    padding: 0.5rem 1rem;
+    color: var(--primary-red);
+    font-weight: 600;
+}
+
+.nav-menu button:hover {
+    color: var(--primary-red);
+}
+
+.material-icons {
+    cursor: pointer;
+    padding: 0.35rem;
+    border-radius: 50%;
+    transition: .2s all;
+}
+
+.nav-menu .material-icons:hover {
+    background-color: var(--white);
+    color: var(--primary-red);
+}
+
+.search-bar input {
+    border: none;
+    outline: none;
+    font-family: inherit;
+    padding: 0.5rem;
+    background-color: transparent;
+    width: 25vw;
+    max-width: 600px !important;
```

Comment:
Vikram: "using `!important` says there is something wrong "

fix `!important` by specificity or source order

Reply:
Thanks for noticing. Fixed.

Reply:
Good to merge